docs/content/mobile/android/local-deploy.md

```diff
@@ -16,6 +16,12 @@ To connect your Android device to a local deployment, you need to prepare its co
 1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
 1. Select **Save** and close the window.
 
+Create and add a MonoBehaviour script to your Android client-worker GameObject.
```

Comment:
this doesn't flow that nicely as it breaks the enumeration. Maybe just remove this sentence?

Reply:
Is this section part of setting up your Unity Editor correctly?
I think it is a separate section, so probably needs this bit of text. Maybe add, "To do this:" to make it clearer.

Reply:
This is a separate section, which sort of falls under the same category needed for a user to prepare their project to connect to a local deployment. Adding, "To do this:" makes this clearer but also helps separates the steps from the previous section to avoid any confusion between the two.